docs/source/NewNetwork/NewNetwork.md

```diff
@@ -0,0 +1,144 @@
+# Setting up a New Network
+
+## Introduction
+
+   The purpose of this document is to describe in some detail the process of building a brand-new Indy Node Network (Network) using 4 Stewards on their own separate nodes.
+   It goes into more details than [Starting a Network](../start-nodes.md).
+   These instructions are intended to be used for a distributed or “production” level environment but can be adapted to be used to build a private network.
+
+   This document is heavily based on [Create New Indy Network](https://docs.google.com/document/d/1XE2QOiGWuRzWdlxiI9LrG9Am9dCfPXBXnv52wGHorNE) and the [Steward Validator Preparation Guide v3](https://docs.google.com/document/d/18MNB7nEKerlcyZKof5AvGMy0GP9T82c4SWaxZkPzya4).
+
+## I. Create Network Governance documents (Optional)
+
+   Network Governance describes the policies and procedures by which your new network will run and be maintained. Here’s an example: [Sovrin Governance Framework](https://docs.google.com/document/d/1K8l5MfXQWQtpT49-FHuYn_ZnRC5m0Nwk)
+
+
+## II. Assign Network Trustees
+   
+   Trustees are the people who manage the network and protect the integrity of the Network Governance.  This includes managing the `auth_rules`.
+
+   For a production Network, at least 3 Trustees representing three different persons are required and more are preferred.  For a test Network one Trustee is required and 3 or more are preferred (all Trustee DID’s may belong to the same user on a test network if needed). 
+
+   Initial Trustees (3 preferred) must create and submit a Trustee DID and Verkey so that the domain genesis file can be built.
+
+   Each trustee has to [instal the `indy-cli`](./CLIInstall.md) and [create a Trustee DID](./CreateDID.md).
+
+   Once the Trustees have created their DID and Verkey give the Trustees access to a spreadsheet like [this one](https://docs.google.com/spreadsheets/d/1LDduIeZp7pansd9deXeVSqGgdf0VdAHNMc7xYli3QAY/edit#gid=0) and have them fill out their own row of the Trustees sheet.  The completed sheet will be used to generate the genesis transaction files for the network.   
+
+
+## III. Genesis Stewards
+
+   A Steward is an organization responsible for running a Node on the Network
+
+   Exactly 4 “Genesis” Stewards are needed to establish the network, more Stewards can be added later.
```

Comment:
I recommend changing this to:
"Exactly 4 “Genesis” Stewards are required to establish the network, but at least 7 are recommended for a production network that you expect to last for a long time. More Stewards can be added later.